x/vmbridge/types/util.go

```diff
@@ -0,0 +1,30 @@
+package types
+
+import (
+	"encoding/json"
+	"fmt"
+	wasmvmtypes "github.com/CosmWasm/wasmvm/types"
+)
+
+func GetWasmVMQueryRequest(requestData []byte) (*wasmvmtypes.QueryRequest, error) {
+	var request wasmvmtypes.WasmQuery
+	if err := json.Unmarshal(requestData, &request); err != nil {
+		return nil, err
+	}
+
+	requestNum := 0
+	if request.Smart != nil {
+		requestNum++
+	}
+	if request.Raw != nil {
+		requestNum++
+	}
+	if request.ContractInfo != nil {
+		requestNum++
+	}
+	if requestNum != 1 {
```

Comment:
thie check is not necessary, because it will use smart query at first, if smart is nil, then raw and then ContractInfo

Reply:
this check will be avoid to make user send 2 quer。it‘s necessary